scripts/test_lib.sh

```diff
@@ -105,14 +105,17 @@ function run_for_module {
 }
 
 function modules() {
-  echo "go.etcd.io/etcd/api/v3
-        go.etcd.io/etcd/pkg/v3
-	go.etcd.io/etcd/raft/v3
-	go.etcd.io/etcd/client/v2
-	go.etcd.io/etcd/client/v3
-	go.etcd.io/etcd/v3
-	go.etcd.io/etcd/etcdctl/v3
-	go.etcd.io/etcd/tests/v3"
+  modules=(
+    go.etcd.io/etcd/api/v3
+    go.etcd.io/etcd/pkg/v3
+	  go.etcd.io/etcd/raft/v3
```

Comment:
The indentation here seems incorrect?

Reply:
Fixed. Tabs vs. spaces. Will need to add a checker to normalize bash scripts around 'spaces'.